core/src/main/java/org/jdbi/v3/core/mapper/MapMapper.java

```diff
@@ -31,8 +31,9 @@
  * keys are column names, while map values are the values in those columns. Map keys are converted to lowercase by
  * default.
  *
- * @see GenericMapMapperFactory for generics support (i.e. to get {@code Map<String, X>})
+ * @deprecated see {@link GenericMapMapperFactory} for generics support (i.e. to get {@code Map<String, T>})
```

Comment:
I'm not sure I agree with deprecating. `Map<String, T>` means that all columns are the same type, but `Map<String, Object>` means the columns could all be different types.

Reply:
Oh yes, that's true